src/view/AvailabilityGraph.js

```diff
@@ -1,30 +1,28 @@
-import { h } from '@cycle/dom'
+import { h } from "@cycle/dom";
 
-function AvailabilityGraph (values = [], range = '720') {
-  const count = +range
-  const height = 120
-  const width = 1440
-  const step = width / count
+function AvailabilityGraph(values = [], range = "720") {
+  const count = +range;
+  const height = 120;
+  const width = 1440;
+  const step = width / count;
 
   const points = values
     .slice(-(1 + count))
-    .reduce((r, v, i) => `${r} ${i * step},${height - v * height}`, '')
+    .reduce((r, v, i) => `${r} ${i * step},${height - v * height}`, "");
 
   const attrs = {
-    class: 'service-graph',
-    viewBox: `0 0 ${width} ${height}`,
-  }
+    class: "service-graph",
+    viewBox: `0 0 ${width} ${height}`
+  };
 
   const lineAttrs = {
-    fill: 'none',
-    stroke: '#757575',
-    'stroke-width': 1,
-    points,
-  }
+    fill: "none",
+    stroke: "#757575",
+    "stroke-width": 1,
+    points
+  };
 
-  return h('svg', { attrs }, [
-    h('polyline', { attrs: lineAttrs })
-  ])
+  return h("svg", { attrs }, [h("polyline", { attrs: lineAttrs })]);
```

Comment:
The newline reduction seems very aggressive here. The call to `h('polyline')` is pretty noisy, syntax-wise—just look at that closing `})])`—so I think it's kind of nice when that's on its own line. Not a dealbreaker, though.